flink/v1.16/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSinkV2.java

```diff
@@ -60,7 +54,7 @@
 import org.junit.runners.Parameterized;
 
 @RunWith(Parameterized.class)
-public class TestFlinkIcebergSinkV2 {
+public class TestFlinkIcebergSinkV2 extends TestFlinkIcebergSinkBase {
```

Comment:
Need to move over the v2 cases to the branch specific test class to validate that row level operations in Flink work as expected on branches